docs/self-managed/connectors-deployment/connectors-configuration.md

```diff
@@ -199,6 +199,38 @@ Package this class and all its dependencies as a JAR, for example `my-secret-pro
 provider implementation. Add this JAR to the runtime environment, depending on your deployment setup.
 Your secret provider will serve secrets as implemented.
 
+To use with [Camunda Helm charts](https://artifacthub.io/packages/helm/camunda/camunda-platform), you can build an init container to create a volume with your secret provider and mount it into the Connectors pod. For example, use the following file as input for your Helm install command:
```

Comment:
```suggestion
To use this JAR with [Camunda Helm charts](https://artifacthub.io/packages/helm/camunda/camunda-platform), build an [init container](https://kubernetes.io/docs/concepts/workloads/pods/init-containers/) to create a volume with your secret provider, and mount it into the Connectors pod. 

For example, use the following file as input for your `helm install` command:
```

Small suggestion; it could be nice to link to the supporting kubernetes docs here, and be a little more direct. If "Helm install command" is referring to `helm install`, the inline code formatting might be more descriptive.